### DISCUSSION TITLE
Is it possible to use letters instead of digits for quick completion?

### DISCUSSION THREAD

**Discussion by tamwile** [company-mode/company-mode#1142] [Q&A] [answered]
I use a non qwerty keyboard, and i have to press shift to type a digit.
This is rather annoying when using company.

Is there a way to use letters instead, like M-a, but this keybind should work as usual when not in company-active-map.

**Comment by yugaego** [comment, marked as answer]
Yes, this is possible and configuration was [simplified recently](https://github.com/company-mode/company-mode/issues/188#issuecomment-877113728).

Now there're two optional ways to configure quick-access keys:

### Customize interface

- Run `M-x customize-group RET company`.
- Set `Company Quick Access Keys` and `Company Quick Access Modifier` according to your needs.

Customization interface suggests some predefined sets of the keys (`Digits` and `QWERTY home row`), but in your case, you should probably choose `User defined` and add the keys one by one.

### Init file

```
(custom-set-variables
 '(company-quick-access-keys '("a" "o" "e" "u" "i"))
 '(company-quick-access-modifier 'super))
```
Note that a modifier should be one of `'meta`, `'super`, `'hyper`, `'control` and you *don't need* to change a modifier if the default `'meta` works fine for you.
